src/project-tests/tribute-page-tests.js

```diff
@@ -60,8 +63,8 @@ export default function createTributePageTests() {
       });
 
       it(`Within the "img-div" element, I should see an element with
-      a corresponding id="img-caption" that contains textual content describing
-      the image shown in "img-div".`,
+      a corresponding id="img-caption" that contains textual content
```

Comment:
@lasjorg Thanks for your contributions. Do you want to include the wording here about the `<figcaption>` in the same way you did in user story 3?

Reply:
Sorry I didn't see this till now.

Just so I'm clear. You mean something like this:

`Within the "img-div" element, I should see either a <figcaption> or <div> element with a corresponding id="img-caption"...`

If so I think that might be a good idea.

Reply:
Well, I forgot to leave a comment, anyway I made the change.